Library/ViewModels/CreatorBylineViewModel.swift

```diff
@@ -23,21 +26,32 @@ public final class CreatorBylineViewModel: CreatorBylineViewModelType,
   CreatorBylineViewModelInputs, CreatorBylineViewModelOutputs {
   public init() {
     let project = self.projectProperty.signal.skipNil()
+    let creatorDetails = self.creatorDetailsProperty.signal.skipNil()
+      .map { $0 }
 
     self.creatorLabelText = project.map {
       Strings.project_creator_by_creator(creator_name: $0.creator.name)
     }
 
     self.creatorImageUrl = project.map { URL(string: $0.creator.avatar.small) }
+    self.creatorStatsText = creatorDetails.map { creatorDetails in
```

Comment:
I think if the launched projects count is `1` this text should emit "First-time creator" for the first part of the string.

Reply:
The ACs changed we are no longer using that copy.

Reply:
correct! we are using "X created • X backed" due to Vicki's recommendation. A slack convo changed the AC's and i tried to update all of the tickets I found to reflect this